ansible/inventories/sandbox/group_vars/inventory-next/vars.yml

```diff
@@ -0,0 +1,53 @@
+---
+# In sandbox, we use python3 on bionic web hosts
+# https://github.com/GSA/datagov-infrastructure-modules/issues/32
+ansible_python_interpreter: /usr/bin/python3
+
+datagov_service: inventory
+app_type: inventory-next
+inventory_db_name: inventory_db_next
+inventory_python_home: "{{ inventory_ckan_python_home }}"
+datagov_in_service: false
+
+
+# Inventory Datapusher
+inventory_datastore_ro_user: datastore_ro
+inventory_datastore_user: datastore
+
+# project path
+project_source_rollback_path: /usr/lib/ckan-rollback
+project_source_new_code_path: /usr/lib/ckan-new
+current_source_symlink: /usr/lib/ckan
+project_index_path: /usr/lib/ckan
+
+# db credentials
+inventory_ckan_db_host: "{{ inventory_next_postgresql_login_host }}"
+inventory_db_user: "{{ inventory_next_postgresql_login_user }}"
+inventory_db_pass: "{{ inventory_next_postgresql_login_password }}"
```

Comment:
It doesn't really matter since we stopped using the datagov-deploy-postgresql role, but the idea here is that postgresql_login_user is the _admin_ user, and then a separate application user would get created (by admin) for use by CKAN.

The advantage being it's easier to rotate credentials for the application (create a new user with new creds without affecting the existing user).

FWIW, in catalog-next staging, I did make this distinction and there is a separate admin user and ckan user. For catalog-next sandbox, the creds are shared.

Reply:
This makes more sense for staging/production, as those machines will stay more static. As we can rapidly deploy sandbox environments, having to build out extra credentials manually will take more time and documentation than is probably valuable in this instance. Noting here though that we will do this for staging and production...